libs/common/src/typegraph/types.rs

```diff
@@ -116,8 +117,16 @@ pub enum StringFormat {
     Ean,
     Date,
     DateTime,
-    // Path,
     Phone,
+    Name,
+    Password,
+    Gender,
+    Firstname,
+    Lastname,
+    Prefix,
+    Profession,
```

Comment:
thought: I've some reservations about adding more here beyond what [json schema](https://json-schema.org/understanding-json-schema/reference/string#built-in-formats) provides. It's a very natural for your implementation but anything you add to the API, someone will depend on and will have to maintained. And unlike the new ones, I believe the rest of the formats have very standard type checking semantics. Localization and more makes it tricky to add any kind of sensible functionality here.

Reply:
@afmika @Natoandro @zifeo what do y'all think?

Reply:
Also, ime, random name and profession and gender generation is rarely used in production but to generate test data. Not sure if that usecase justifies adding this capability to the typegraph.

Reply:
Makes sense, I believe we should keep this as simple as possible as it is mostly intended for testing. We should avoid any locale specific generators.

Reply:
Also, things like genders can always be described with a `t.enum(...)`.

Reply:
Ok, I will wait for some reviews and push an update.

Reply:
Yeah. I agree with @Yohe-Am on this.

Reply:
sure @Natoandro, I will make the changes.

Reply:
Yeah, any idea what direction you're going to take it @destifo ? The current impl seems to be appropriate in a lot of ways.

Reply:
A solution might be to restrict these features/semantics just to the `from_random` feature. I.e. args on `from_random`. It's still technically becomes part of the typegraph spec though.

Reply:
so these features are still possible with no need to provide any arguments to the `from_random` method, as you can add them by using the `config` parameter(check the `random/random_injection/random_injection.py` test) in the typenode definition. I added these **StringFormats** to add extra flexibility to the user. so the direction I am thinking of right now is to remove the newly added StringFormats because generating random values of different types is still possible using `config` parameter of the typenode. what do you think @Yohe-Am.